server/@types/express/index.d.ts

```diff
@@ -36,7 +36,8 @@ export declare global {
     }
 
     interface Locals {
-      user: Express.User
+      user: PrisonUser
+      feComponentsMeta?: HeaderFooterMeta
```

Comment:
The DPS components module (with `includeMeta: true`) adds data to `res.locals`. Defining here so it is typed correctly.